BUILDING.md

```diff
@@ -318,36 +318,41 @@ If you are updating tests and want to run tests in a single test file
 (e.g. `test/parallel/test-stream2-transform.js`):
 
 ```text
-$ tools/test.py test/parallel/test-stream2-transform.js
+$ python3 tools/test.py test/parallel/test-stream2-transform.js
 ```
 
+> Note: Using `pyhton3` alias is necessary only if you don't have
+> a python executable on your path.
+
```

Comment:
```suggestion
```

Ideally, you would need to use `python3 ...` explicitly only on Windows, where the `#!/usr/bin/env python` shebang doesn't work but in this case `tools/test.py` doesn't work on any platform because it's actually a python3 script and `python` = `python2` wherever it exists AFAIK.

<details>

<summary>error</summary>

```sh
$ tools/test.py
  File "tools/test.py", line 230
    print(f" failed {len([i for i in outputs if i.UnexpectedOutput()]) + 1} out of {self.measure_flakiness + 1}")
                                                                                                               ^
```

</details>

Reply:
Nice catch.